exercise1/exercise4/main.cpp

```diff
@@ -0,0 +1,20 @@
+//Exercise 4 - Underflow and Overflow
+//Write a program that will illustrate overflowand underflow.
+//Hint: Use the limits from the previous exercise and try adding to the max limit, or subtracting from the min limit.
+
+
+//Diego Guerrero
+#include <iostream>
+
+int main() {
+
+	int overflow = SHRT_MAX + 1;
+	int underflow = SHRT_MIN - 1;
+
+	std::cout << "��Esto pasa cuando no cuidas tus datos!!" << std::endl;
+
+	std::cout << "---------------------------------------" << std::endl;
+
+	std::cout << "El overflow sucede cuando usas una variable 'short' y el valor que utilizas es (" << SHRT_MAX << ")+1 \n El valor seria=" << overflow << std::endl;
```

Comment:
Test your code and make sure that it's working the way it should. There is no overflow or underflow going on here, ints can definitely store SHRT_MAX + 1 and SHRT_MIN - 1 without a problem.